QRCoder.Xaml/XamlQRCode.cs

```diff
@@ -1,5 +1,4 @@
-﻿#if NETFRAMEWORK || NET5_0_WINDOWS || NET6_0_WINDOWS
```

Comment:
Can you explain, why you removed the constants without replacement? :-)

Reply:
This project only targets these tfms and so this condition is always true.

I didn't really look much at the other projects, to be honest.  I just focused on the main project and the test project.